### PR TITLE
chore(logs): Add deis logs test

### PR DESCRIPTION
**Pull request by jchauncey**

### CODE REVIEW

File: tests/apps_test.go [hunk 1]
```diff
@@ -177,12 +177,11 @@ var _ = Describe("deis apps", func() {
 
 			// Pending: The logging subsystem is not currently installed as part of the workflow-dev
 			// chart.
```
Comment (krancour) [inline tests/apps_test.go]: Should remove this comment.

File: tests/apps_test.go [hunk 2]
```diff
@@ -175,14 +175,11 @@ var _ = Describe("deis apps", func() {
 				Eventually(sess).Should(Exit(0))
 			})
 
-			// Pending: The logging subsystem is not currently installed as part of the workflow-dev
-			// chart.
-			XSpecify("that user can retrieve logs for that app", func() {
+			Specify("that user can retrieve logs for that app", func() {
 				sess, err := cmd.Start("deis logs -a %s", &user, app.Name)
```
Comment (arschles) [inline tests/apps_test.go]: not too familiar with this `Start` func, so ignore me if I'm mistaken, but is the first parameter supposed to be a format string? if so, there are not enough format directives (i.e. `%s`) for the number of params given

Comment (jchauncey) [inline tests/apps_test.go]: `start` takes the user running the command as the second param.

Comment (arschles) [inline tests/apps_test.go]: Nevermind, after talking privately I understand that this runs the command as the given used, with `app.Name` as the single param to the format string

**Pull request merged**